projects/court-case-and-delius/src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/ProxyIntegrationTest.kt

```diff
@@ -313,5 +319,3 @@ internal class ProxyIntegrationTest {
         assertThat(res.totalNumberOfRequests, equalTo(1))
     }
 }
-
-//{"status":403,"developerMessage":"This is a restricted record. Please contact a system administrator"}
```

Comment:
Can this be removed?

Reply:
It has been removed - its commented that i have removed

Reply:
Doh sorry I always make that mistake